pkgs/tools/text/gawk/default.nix

```diff
@@ -57,7 +68,12 @@ stdenv.mkDerivation rec {
 
   inherit doCheck;
 
-  postInstall = ''
+  postInstall = (if interactive then ''
+    remove-references-to -t "$NIX_CC" -- "$out"/bin/gawkbug
+    patchShebangs --host -- "$out"/bin/gawkbug
+  '' else ''
+    rm -- "$out"/bin/gawkbug
```

Comment:
Also FYI: we don't need -- here because $out always starts with a slash.

Reply:
Fixed.

Note: I don’t think that’s documented in Nix (i.e. that output paths are guaranteed to be absolute paths)…